gix-blame/src/file/tests.rs

```diff
@@ -1338,3 +1338,100 @@ mod process_changes {
         );
     }
 }
+
+mod blame_ranges {
```

Comment:
Can we add tests for the following cases:

- `vec![0..=x]`
- `vec![1..=3, 5..=7 ,2..=6]` (this is the one also mentioned in another comment)